File: Codigo_Fonte_c/Lista 3/lista3_8.c
#include <stdio.h>

int main(){
	float nota_turma[80];
	int i;
	float mediaTurma, sum_nota = 0 ;
	for (i = 0; i < 80; i++){
		scanf("%f", &nota_turma[i]);
	}
	for (i = 0; i < 80; i++){
		sum_nota = sum_nota + nota_turma[i];
	}
	mediaTurma = sum_nota / 80;
	printf("%.2f\n",mediaTurma);
	return 0;	
}

File: Codigo_Fonte_pascal/lista3/lista3_8.pas
program media_turma;

var
   nota_turma : array[1..80] of real;
   i : integer;
   media_turma, sum_nota :real;

begin
     for i := 1 to 80 do
     begin
          readln(nota_turma[i]);
          sum_nota := sum_nota + nota_turma[i];
     end;

     media_turma := sum_nota / 80;
     writeln(media_turma);
     readln;
end.
